src/Storage/ContentRequest/Edit.php

```diff
@@ -108,6 +108,18 @@ public function action(Content $content, array $contentType, $duplicate)
             $contentowner = $this->users->getUser($content->getOwnerid());
         }
 
+        // Build list of incoming non inverted related records.
+        $incomingNotInverted = [];
+        foreach ($content->getRelation()->incoming($content) as $relation) {
+            if (!$relation->isInverted()) {
+                $record = $this->em->getContent($relation['from_contenttype'] . '/' . $relation['from_id']);
+
+                if ($record) {
+                    $incomingNotInverted[$relation['from_contenttype']][] = $record;
+                }
+            }
+        }
+
```

Comment:
The ArrayAccess is for BC and really should be avoided internally (not to mention the nesting)… Try:
```php
        // Build list of incoming non inverted related records.
        $incomingNotInverted = [];
        /** @var Relations $relation */
        foreach ($content->getRelation()->incoming($content) as $relation) {
            if ($relation->isInverted()) {
                continue;
            }
            $fromContentType = $relation->getFromContenttype();
            $record = $this->em->getContent($fromContentType . '/' . $relation->getFromId());
            if ($record) {
                $incomingNotInverted[$fromContentType][] = $record;
            }
        }
```